rest-service/manager_rest/storage/resource_models.py

```diff
@@ -420,6 +420,7 @@ class Filter(CreatedAtMixin, SQLResourceBase):
     __tablename__ = 'filters'
 
     value = db.Column(JSONString, nullable=True)
+    updated_at = db.Column(UTCDateTime)
```

Comment:
I don't think there is a need to also add `updated_by`. What do you think?

Reply:
we already have several `updated_at` but there's no `updated_by`

Reply:
Yep, same as I thought.